### PR TITLE
1810 clean up data viz UI component

### PR DESCRIPTION
**Pull request by heyitsmebev**
*A note to PR reviewers: it may be helpful to review our [code review documentation](https://github.com/GSA/notifications-api/blob/main/docs/all.md#code-reviews) to know what to keep in mind while reviewing pull requests.*

## Description

This PR is to remove socket io and to fetch the data the usual way. I uninstalled socket and removed the code and testing related. 

The original problem:

- A string of 400 API responses
- 7 day chart not loading

### CODE REVIEW

**Review by ccostino — COMMENTED** [review]
This all looks good to me, thanks @heyitsmebev!

One quick question - do we have to remove anything related to SocketIO on the Python side at all, either in the admin and/or in the API?

File: app/assets/javascripts/activityChart.js [hunk 1]
```diff
@@ -185,41 +185,38 @@
...
-
-            var labels = [];
-            var deliveredData = [];
-            var failedData = [];
+        var daily_stats = activityChartContainer.getAttribute('data-daily-stats');
+        var daily_stats_by_user = activityChartContainer.getAttribute('data-daily_stats_by_user');
+
+        try {
+            var serializedStats = type === 'service' ? daily_stats : daily_stats_by_user;
+            serializedStats = serializedStats.replace(/'/g, '"');
```
Comment (terrazoon) [inline app/assets/javascripts/activityChart.js:193]: This replace is a little mysterious.  maybe a comment?

**Review by terrazoon — APPROVED** [review]

**Comment by heyitsmebev** [comment]
@ccostino you're right! I removed flask-socketio

**Review by ccostino — APPROVED** [review]
Thanks, @heyitsmebev!

**Pull request merged**